tests/core/no-netplan-default-config/task.yaml

```diff
@@ -34,10 +34,8 @@ execute: |
     networkctl status $eth_if | MATCH '(unmanaged|failed)'
     network-manager.nmcli c show --active | MATCH $eth_if | MATCH -v netplan
 
-    # Verify the active connections
-    test `network-manager.nmcli -m multiline -f UUID c show --active | wc -l` -eq 2
-    network-manager.nmcli -m multiline -f DEVICE c show --active | MATCH "lo"
-    network-manager.nmcli -m multiline -f DEVICE c show --active | MATCH "ens4"
+    # We should only have a single active configuration
+    test `network-manager.nmcli -m multiline -f UUID c show --active | wc -l` -eq 1
```

Comment:
I am surprised we still use backtick for command substitution.

Reply:
This was indeed a revert of an older commit I had done, but I can change this to subshell